### PR TITLE
[api] document publish status api call

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

File: docs/api/api/api.txt [hunk 1]
```diff
@@ -1648,6 +1648,13 @@ GET /published/<project>/<repository>
 XmlResult: directory
 
 
+GET /published/<project>/<repository>?view=status
+
+  Delivers last publish information like time and repository build id
+
+XmlResult: publish_status
```
Comment (hennevogel) [inline docs/api/api/api.txt]: Did you forget to commit this file?

Comment (adrianschroeter) [reply docs/api/api/api.txt]: no, I was too lazy to create it. reference to buildstatus existed also before without the definition (like many others)

**Comment by adrianschroeter** [comment]
@Conan-Kudo fixed ... (new notebook)

File: docs/api/api/api.txt (hunk 1 above)
Comment (hennevogel) [reply docs/api/api/api.txt]: ok

**Pull request merged**